styles/variables-silicon.less

```diff
@@ -152,18 +152,25 @@
 // Popup
 // ---------------------------------------
 @agate-popup-min-width: 510px;
-@agate-popup-max-width: 70%;
 @agate-popup-border-width: 1px 0;
-@agate-popup-border-radius: 20px;
-@agate-popup-border: 1px solid transparent;
-@agate-popup-padding: (@agate-component-spacing * 4) @agate-component-spacing;
-@agate-popup-title-padding: 0 (@agate-button-small-height) 0 0;
-@agate-popup-body-padding: (@agate-component-spacing * 2);
-@agate-popup-body-margin: -@agate-component-spacing;  // Extend to the edge
-@agate-popup-body-border-radius: 0;
-@agate-popup-buttons-padding: (2 * @agate-component-spacing) 0 0 0;
+@agate-popup-border-radius: 39px;
+@agate-popup-border: none;
+@agate-popup-padding: (@agate-component-spacing * 2) 0 0 0;
+@agate-popup-closebutton-title-padding-left: @agate-component-spacing;
+@agate-popup-closebutton-title-padding-right: @agate-button-small-height;
+@agate-popup-title-padding: 0 @agate-component-spacing;
+@agate-popup-body-padding: (@agate-component-spacing * 2) @agate-component-spacing 0 @agate-component-spacing;
+@agate-popup-body-margin: 0;  // Extend to the edge
+@agate-popup-body-border-radius: 0 0 @agate-popup-border-radius @agate-popup-border-radius;
+@agate-popup-buttons-border-radius: @agate-popup-body-border-radius;
+@agate-popup-buttons-padding: @agate-component-spacing 0;
+@agate-popup-content-padding: @agate-component-spacing @agate-component-spacing (3 * @agate-component-spacing) @agate-component-spacing;
 @agate-popup-closebutton-position-left: auto;
 @agate-popup-closebutton-position-right: 0;
+@agate-popup-opacity: 0.95;
```

Comment:
The entire popup is 95% opacity? even the content? What if we move this to just the background color? `fade(@bgcolor, 95%)`